usr/share/ublue-os/just/custom.just

```diff
@@ -169,7 +170,6 @@ podmansh-switch IMAGE:
   systemctl --user start podmansh.service
   echo "Image now switched to {{IMAGE}}"
 
-# Install better touch-friendly GNOME extensions
```

Comment:
Lets leave in the just description for `just touch` command

Reply:
I'll add this back in in a new PR